composer.json

```diff
@@ -28,7 +28,7 @@
         "ext-zip": "*",
         "api-platform/core": "^3.4",
         "beberlei/doctrineextensions": "^1.0",
-        "bjeavons/zxcvbn-php": "^1.4",
+        "bjeavons/zxcvbn-php": "^1.4.2",
```

Comment:
You don't really need to update the json here, the `^1.4` includes all the patches, and just running `composer update bjeavons/zxcvbn-php` would work to update the lock file
(not blocking though)